### PR TITLE
change kk.py to relative import of data.py; update deprecated numpy.i…

### PR DESCRIPTION
**Pull request by delongchamp**
…nt to numpy.int32 in kk.py

### CODE REVIEW

**Comment by delongchamp** [comment]
Hi Ben, we use this repo all the time on our RSoXS team in headless mode / Jupyter notebooks. This is a quick fix to a fatal error on deprecated np.int.

File: kkcalc/data.py [hunk 1]
```diff
@@ -377,7 +377,7 @@ def calculate_asf(Stoichiometry):
 		total_E = numpy.unique(total_E)
 		# add weighted asf data sets for KK calculation
 		total_Im_coeffs = numpy.zeros((len(total_E)-1, 5))
-		counters = numpy.zeros((len(Stoichiometry)),dtype=numpy.int)
+		counters = numpy.zeros((len(Stoichiometry)),dtype=numpy.int32)
```
Comment (benajamin) [inline kkcalc/data.py:380]: I don't think it is good practice to specify the size of the integer here, does it work OK for you if the dtype is set to just `int`?

Comment (delongchamp) [reply kkcalc/data.py:380]: Thanks Ben, that should be fine! This change was made in response to deprecation of numpy.int, which was always just an alias for native Python int.  I believe this should work fine if it's just dtype = int.

Comment (delongchamp) [reply kkcalc/data.py:380]: Note also the deprecation of numpy.bool.

**Pull request merged**